code/game/gamemodes/cult/runes.dm

```diff
@@ -313,8 +311,10 @@ structure_check() searches for nearby cultist structures required for the invoca
 
 	rune_in_use = TRUE
 	var/mob/living/L = pick(offer_targets)
-	if(L.mind in GLOB.sacrificed)
-		fail_invoke()
+	if(HAS_TRAIT(L, TRAIT_CULT_IMMUNITY))
+		fail_invoke(FALSE)
+		for(var/I in invokers)
+			to_chat(I, "<span class='warning'>This sacrifice was already converted recently. Wait a minute before trying again!</span>")
```

Comment:
what exactly are the types of I in this case?

Reply:
invokers is a list of the cultists who either clicked on the rune or were standing around it. That list is made in ``can_invoke()`` apparently so the ``type is /mob/living.``